tests/EmailValidator/Validation/RFCValidationTest.php

```diff
@@ -278,17 +279,29 @@ public function getInvalidEmailsWithWarnings()
                 'example@toolonglocalparttoolonglocalparttoolonglocalparttoolonglocalpart.co.uk'
             ],
             [
-                [DomainTooLong::CODE, LabelTooLong::CODE,],
+                [DomainTooLong::CODE, LabelTooLong::CODE, EmailTooLong::CODE],
                 'example2@toolonglocalparttoolonglocalparttoolonglocalparttoolonglocalparttoolonglocalparttoolonglocal'.
                 'parttoolonglocalparttoolonglocalparttoolonglocalparttoolonglocalparttoolonglocalparttoolonglocalpart'.
                 'toolonglocalparttoolonglocalparttoolonglocalparttoolonglocalpart'
             ],
             [
-                [DomainTooLong::CODE, LabelTooLong::CODE,],
+                [DomainTooLong::CODE, LabelTooLong::CODE, EmailTooLong::CODE],
                 'example@toolonglocalparttoolonglocalparttoolonglocalparttoolonglocalparttoolonglocalparttoolonglocal'.
                 'parttoolonglocalparttoolonglocalparttoolonglocalparttoolonglocalparttoolonglocalparttoolonglocalpart'.
                 'toolonglocalparttoolonglocalparttoolonglocalparttoolonglocalpar'
             ],
+            [
+                [LabelTooLong::CODE,],
+                sprintf('example@%s.com', str_repeat('ъ', 60)),
```

Comment:
@vajexal while porting to v3, why this should rise the warning? it is a 60 char label long

Reply:
because if we convert label to idn, it's length will be > 63

```php
var_dump(idn_to_ascii(str_repeat('ъ', 60), IDNA_DEFAULT, INTL_IDNA_VARIANT_UTS46, $idnaInfo), $idnaInfo);
// bool(false)
// array(3) {
//   ["result"]=> string(66) "xn--z1aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
//   ["isTransitionalDifferent"]=> bool(false)
//   ["errors"]=> int(2)
// }
```